targetconfig.json

```
@@ -5,11 +5,20 @@
             "LEGO"
         ],
         "approvedRepos": [
-            "microsoft/pxt-automation"
+            "microsoft/pxt-automation",
+            "THEb0nny/makecode-ev3-hitechnic-color-sensor-v2"
         ],
         "preferredRepos": [
             "microsoft/pxt-automation"
-        ]
+        ],
+        "approvedRepoLib": {
+            "microsoft/pxt-automation": {
+                "preferred": true
+            },
+            "THEb0nny/makecode-ev3-hitechnic-color-sensor-v2": {
```

```suggestion
            "theb0nny/makecode-ev3-hitechnic-color-sensor-v2": {
```

slug needs to be lower cased for this field, since we lowercase before indexing into it. I think build should catch this, I just kicked it off~